tox.ini

```diff
@@ -22,7 +22,11 @@ changedir =
     docs: doc
 deps =
     dev: numpy>=0.0.dev0
+    dev: scipy>=0.0.dev0
     dev: astropy>=0.0.dev0
+    dev: pandas>=0.0.dev0
+    dev: git+https://github.com/radio-astro-tools/casa-formats-io.git
```

Comment:
Can you double check that v0.3.0 works now?

Reply:
I've tried to pip install on macOS several times now, always getting a numpy 1.26 version

Reply:
Maybe a caching problem just as with shapely; seems to be running here as well, but then the installed versions of other deps don't make any sense with what is in `tox.ini` now. Rebasing to try to get this sorted...